### ISSUE TITLE
Add the possiblity to search for packages registered in cpm.rocks

### ISSUE THREAD

**Issue by cogumbreiro** [iauns/cpm#11]
It is very difficult to search for registered packages, basically amounts to using Google.

**Comment by sakeel** [comment]
+1. Package browsing for this package manager sucks and makes it difficult to use the project. If you want more adoption, that would be a good place to start. Just an alphabetic list would be sufficient to start with.

**Comment by iauns** [comment]
I agree. I like your idea of producing a simple alphabetical list of all packages currently registered. It should make browsing a bit less painful. I'll look in to this and also into open sourcing the website.

**Comment by 0xPr0xy** [comment]
+1

**Comment by ekg** [comment]
:+1: 

Search would really help motivate the use of CPM.

**Comment by ofan** [comment]
It's been over 6 months now, is there any news about this issue? I found it very difficult to locate the package I want using cpm.rocks at the moment.

**Comment by tomq42** [comment]
Vital functionality.

**Comment by xvapx** [comment]
+1

**Comment by sledgeh** [comment]
https://duckduckgo.com/?q=boost+site%3Acpm.rocks
Change `boost` with what you need. At this moment this is a solution.
